src/ralph/cli.py

```diff
@@ -852,7 +834,7 @@ def runserver(backend: str, host: str, port: int, **options):
             if value is None:
                 continue
             backend_name, field_name = key.split(sep="_", maxsplit=1)
-            key = f"RALPH_BACKENDS__LRS__{backend_name}__{field_name}".upper()
+            key = f"RALPH_BACKENDS__DATA__{backend_name}__{field_name}".upper()
```

Comment:
The `fs` lrs backend has a specific settings class (with field `DEFAULT_LRS_FILE`) that would be ignored after this change, no?
Should we expect more specific settings for the lrs backends? In this case we could probably overwrite the `Config` class `env_prefix` attribute to `"RALPH_BACKENDS__LRS__"`, and revert this line here

Reply:
Good point) Thanks) It would indeed be better to keep the data backend settings separated from the lrs backend settings) Updated the `env_prefix` of lrs backends)